src/resources/language/_language-pt.yml

```diff
@@ -51,3 +51,22 @@ crossref-eq-prefix: "Equação"
 crossref-lof-title: "Lista de Figuras"
 crossref-lot-title: "Lista de Tabelas"
 crossref-lol-title: "Lista de Listagens"
+
+listing-page-order-by: "Ordenar por"
+listing-page-order-by-default: "Pré-selecionado"
+listing-page-order-by-date-asc: "O mais novo"
```

Comment:
Does `date-asc` show newest-date first? That's what the translation implies but it feels to me that "date ascending" would be oldest-date first.

Reply:
No you're right the strings are reversed. Will fix!